client/blocks/reader-post-card/index.jsx

```diff
@@ -2,30 +2,31 @@
  * External Dependencies
  */
 import React, { PropTypes } from 'react';
-import { noop, truncate, trim, get } from 'lodash';
+import { noop, truncate } from 'lodash';
 import classnames from 'classnames';
 import ReactDom from 'react-dom';
 import closest from 'component-closest';
 
 /**
  * Internal Dependencies
  */
-import AutoDirection from 'components/auto-direction';
 import Card from 'components/card';
 import DisplayTypes from 'state/reader/posts/display-types';
-import ReaderPostActions from 'blocks/reader-post-actions';
 import * as stats from 'reader/stats';
+import ReaderPostActions from 'blocks/reader-post-actions';
 import PostByline from './byline';
-import FeaturedVideo from './featured-video';
-import FeaturedImage from './featured-image';
+import GalleryPost from './gallery';
+import PhotoPost from './photo';
+import StandardPost from './standard';
 import FollowButton from 'reader/follow-button';
-import PostGallery from './gallery';
-import PostPhoto from './photo';
+import PostStoreActions from 'lib/feed-post-store/actions';
 import DailyPostButton from 'blocks/daily-post-button';
 import { isDailyPostChallengeOrPrompt } from 'blocks/daily-post-button/helper';
-import * as DiscoverHelper from 'reader/discover/helper';
+import { getDiscoverBlogName,
+	getSourceFollowUrl as getDiscoverFollowUrl,
+	isDiscoverPost as isDiscover
```

Comment:
I see we're doing this to avoid a naming collision with the const below, but could we leave the function named `isDiscoverPost` and call the const `isDiscover` instead?